### PR TITLE
Add Chrome extensions and local files to Sentry URL blacklist

### PR DESCRIPTION
**Pull request by ZhangYiJiang**
As per title.

### CODE REVIEW

File: www/src/js/bootstrapping/sentry.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ if (loadRaven) {
       // Chrome extensions
       /extensions\//i,
```
Comment (li-kai) [inline www/src/js/bootstrapping/sentry.js]: We could just remove the s here...

Comment (ZhangYiJiang) [reply www/src/js/bootstrapping/sentry.js]: I'm actually a little worried that regex is a little too permissive

File: www/src/js/bootstrapping/sentry.js [hunk 2]
```diff
@@ -14,9 +14,11 @@ if (loadRaven) {
       'canvas.contentDocument',
     ],
     ignoreUrls: [
-      // Chrome extensions
-      /extensions\//i,
+      // Local file system
+      /^file:\/\//i,
+      // Chrome and Firefox extensions
```
Comment (yangshun) [inline www/src/js/bootstrapping/sentry.js]: How does this exclude Firefox extensions? They seem Chrome-specific to me.

Comment (ZhangYiJiang) [reply www/src/js/bootstrapping/sentry.js]: `chrome://` I believe is used internally by both Firefox and Chrome. Also added `moz-extension://` to the list.

**Pull request merged**